Cargo.toml

```diff
@@ -53,6 +54,8 @@ url = { version = "2", optional = true }
 actix-web = { version = "4", optional = true }
 actix-cors = { version = "0.6", optional = true }
 
+# nvtxw:
+nvtxw = { git = "https://github.com/NVIDIA/NVTX.git", branch = "nvtxw-rs", optional = true }
```

Comment:
One thing I just wanted to check is whether this is a long-term stable branch. I'm not sure what would happen if we were to merge this here and then you deleted the upstream branch. Might be better to merge the Rust code upstream before we commit this on our end.

Reply:
Yes, it is a long-term stable branch.

Reply:
Please hold that thought while we discuss this.

Reply:
We are still determining what our final course of action will be, and are trying to arrive at a conclusion ASAP. For the moment, if you could continue to approve CI runs if/when I push an update to the branch, that would be crucially helpful.

Reply:
We have decided to publish the nvtxw crate to crates.io as v0.1.0. As of today it is basically ready, pending one last CI verification. Once we publish it I will update this branch to reference the actual crate and remove this PR's draft status.

Reply:
Crate published, branch updated, marked ready for review.

Reply:
@evanramos-nvidia @tcourtneynv CI is passing, let me know if you're ready to merge.

Reply:
Yes, we are ready for merge.

Reply:
Great! I've updated the Legion MR.